SU-HSDO/suhumsci
# HSD8-192 configuring archaeology subtheme

# READY FOR REVIEW

# Summary
- Create and configure subtheme for Archaeology (hopefully a model to follow for future site that require custom styles)
- Added a new screenshot.png file for the SU HumSci theme and the Archaeology theme
- Removed custom Archaeology styles from the SU HumSci theme
- Question: do we need to set the Archaeology theme as the new default in code for Archaeology?

# Needed By (Date)
- End of sprint (8/16)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Go to `admin/appearance` and verify that new screenshots are rendering and that the Stanford HumSci Archaeology Custom Theme is an option
3. Enable the new theme and set as default
4. Ensure that enabling this theme makes it so that your local site matches the current prod site (accent dashes are a good thing to look at)

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/HSD8-192

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
